core/less/buttons/variables.less

```diff
@@ -1,41 +1,37 @@
-@import "../colors/variables.less";
+@import "../variables.less";
 
 /*
     Buttons variables
     =================
 
-    Type
-    ----
-*/
-
-    @btn-types: 'primary', 'secondary', 'tertiary';
-
-/*
     Color
     -----
 */
-	@btn-colors: 'lime', 'orange', 'soil', 'pool';
-    @btn-darken-hover-factor: 10%;
-	@btn-darken-active-factor: 15%;
+    @btn-colors: 'lime', 'orange', 'soil', 'pool';
```

Comment:
Actually it was on purpose, to show that it can be an environment color, but with no proper button associated. I mean,  `.env-sienna .btn-env` will be of sienna color, but there is no `btn-sienna` class.

If we need it it's easy to add it.